pydoctor/themes/readthedocs/fonts/book.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
it this really a font.. or just a svg ?

Maybe use https://fontawesome.com/ or something similar for icons

Reply:
It's not a font, And i wanted only this SVG, not all the other icons from https://fontawesome.com/. So this is the way I choose to do it.

Reply:
sure. no.

maybe have the file located into `pydoctor/themes/readthedocs/images/` or something similar :)

Reply:
We use fonts for the other theme as well, and I think it's a pretty common name for such a folder.